src/vdaf/prio3.rs

```diff
@@ -143,13 +144,27 @@ fn check_num_aggregators(num_aggregators: u8) -> Result<(), VdafError> {
 }
 
 /// The base type for prio3.
+#[derive(Derivative)]
+#[derivative(Debug)]
 pub struct Prio3<T: Type, A> {
     num_aggregators: u8,
     suite: Suite,
     typ: T,
+    #[derivative(Debug = "ignore")]
     phantom: PhantomData<A>,
 }
 
+impl<T: Type, A> Clone for Prio3<T, A> {
```

Comment:
We can't derive Clone because of the `PhantomData<A>` member, since `A` is not required to be `Clone`, and evidently the derived `Clone` implementation doesn't understand that it doesn't need to work with an actual `A` value.

Reply:
We could avoid the `A` generic by merging `Prio3Result` and `Prio3ResultVec` into `Prio3ResultVec`, if that would simplify things. I thought flexibility would be useful here, but perhaps not if it comes at the cost of complexity.